servers/src/test/java/tachyon/worker/block/evictor/EvictorTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block.evictor;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.BlockStoreLocation;
+import tachyon.worker.block.meta.StorageDir;
+import tachyon.worker.block.meta.StorageTier;
+
+/**
+ * This is a parameterized unit test for all types of {@link Evictor} defined in {@link EvictorType}
+ *
+ * It tests general evictor behavior like not evicting any space when space request is already
+ * available, evicting enough space when eviction is needed, having a null eviction plan when the
+ * requested space can not be satisfied anyway and so on.
+ *
+ * Behavior for a specific type of evictor will be tested in other classes, e.x. tests to ensure
+ * that blocks evicted by LRUEvictor are in the right order should be in LRUEvictorTest.
+ */
+@RunWith(Parameterized.class)
+public class EvictorTest {
+  private static final int USER_ID = 2;
+  private static final long BLOCK_ID = 10;
+  private static final int TEST_TIER = 0;
```

Comment:
`TEST_TIER_LEVEL` to be more clear? It is pretty bad that we have both concept of level and aliasValue---both in int---to keep backwards compatibility. We plan to fix it in the future. But for now, let's be very specific that the int value is when it refers to a tier, so we can fix all references easier later. Thanks